src/main/scala/com/mozilla/telemetry/views/CrashSummaryView.scala

```diff
@@ -119,6 +120,7 @@ case class CrashSummary (
     build_version: String,
     build_id: String,
     channel: String,
+    crash_time: String,
```

Comment:
How about:
```suggestion
    crash_time: Option[String],
```
to keep this consistent with `client_id`?